src/Spice86.Core/Emulator/InterruptHandlers/Timer/TimerInt8Handler.cs

```diff
@@ -42,6 +43,7 @@ public TimerInt8Handler(IMemory memory, IFunctionHandlerProvider functionHandler
     /// <inheritdoc />
     public override void Run() {
         TickCounterValue++;
+        _emulationLoopRecall.RunInterrupt(0x1C);
```

Comment:
I'm not fond of emulationloopRecall, which is essentialy a hack inherited from DOSBox that I implemented.

Why not just call the INT1C Run() method directly ?